docs/UserGuide.md

```diff
@@ -294,6 +294,21 @@ The completion status of each student in the list will be displayed as well.
 
 <br>
 
+### View the list of students by the number of undone tasks in descending order: `sort`
+
+Sorts and displays the students in TAPA by the number of undone tasks in descending order.
+
+**Format**: `sort`
+
+* Displays the students from the list of students by the number of undone tasks in descending order
+* The students are indexed as 1, 2, 3, ......
+
+**Example**:
+* `sort`
+    * Displays all the enrolled students by the number of undone tasks in descending order
```

Comment:
I think it might be a good idea to bold the word "descending" as well.

Reply:
Alright, I will bold "descending" to make it clearer for the user.